src/pages/index.js

```diff
@@ -50,51 +82,132 @@ const editModal = new ModalWithForm(
 
 editModal.setEventListeners();
 
+function handleProfileFormSubmit(data) {
+  editModal.setLoading(true);
+  api
+    .setUserInfo(data)
+    .then((res) => {
+      userInfo.setUserInfo(res);
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      editModal.setLoading(false);
+    });
+  editModal.close();
+}
+
+// Add Card Modal
 const newCardModal = new ModalWithForm(
   {
     modalSelector: "#card__add-modal",
   },
   handleCardFormSubmit
 );
-newCardModal.setEventListeners();
 
-const userInfo = new UserInfo({
-  nameSelector: ".profile__title",
-  descriptionSelector: ".profile__description",
-});
+newCardModal.setEventListeners();
 
-function fillProfileForm() {
-  const info = userInfo.getUserInfo();
-  profileTitleInput.value = info.name;
-  profileDescriptionInput.value = info.description;
+function handleCardFormSubmit(data) {
+  newCardModal.setLoading(true);
+  api
+    .addCard(data)
+    .then((res) => {
+      const card = new Card(
+        res,
+        "#card__template",
+        handleImageClick,
+        handleDeleteClick,
+        handleLikeClick
+      );
+      cardList.prepend(card.getCard());
```

Comment:
It's highly recommended to make a separate function `createCard` for creating a card not to copy-paste the code